rakefile_common.rb

```diff
@@ -156,6 +156,17 @@ def instructions(key)
   @instructions[key]
 end
 
+def sources
+  @sources ||= begin
+    # Make sure all instructions have a `type`
+    if (no_type = instructions(:sources).find { |src| src[:type].to_s.empty? })
+      raise "Missing `type` in #{no_type} file"
+    end
+
+    instructions(:sources).map { |s| Source::Base.instantiate(s) }
```

Comment:
Ah yes, we did discuss this but I forgot to make the change! Fixed in https://github.com/everypolitician/everypolitician-data/commit/3a15a9f8608030b7957918246a78b6911cc009b6.